.github/ISSUE_TEMPLATE/bug-performance-report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug/Performance report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Screenshots**
```

Comment:
Should we ask to paste code / output snippets as pasting images will make it very heavy and non-searchable.

Reply:
I agree to remove screenshots section. I think https://github.com/tensorflow/tensorflow/blob/master/.github/ISSUE_TEMPLATE/00-bug-performance-issue.md has been good enough. System information and minimal reproducable codes are most important in my opinion.

Reply:
Okay i will remove this